project/src/js/ui/clock/clockView.js

```diff
@@ -0,0 +1,68 @@
+import h from '../helper';
+import layout from '../layout';
+import i18n from '../../i18n';
+import clockSettings from './clockSettings';
+import { formatTimeinSecs } from '../../utils';
+import sound from '../../sound';
+
+export default function view(ctrl) {
+  window.StatusBar.hide(); // Put this here instead of ctrl so it is reapplied after a phone lock and unlock
```

Comment:
But here is it called on each redraw, which is not a good thing. You can simply listen to the cordova 'resume' event to hide again the status bar.

You'll need to subscribe to the event and not forget to unsuscribe on unload so the behavior don't apply to other screens.